Formula/musikcube.rb

```diff
@@ -0,0 +1,40 @@
+class Musikcube < Formula
+  desc "Terminal-based audio engine, library, player and server"
+  homepage "https://musikcube.com"
+  url "https://github.com/clangen/musikcube/archive/0.99.6.tar.gz"
+  sha256 "739375d47414ad1d8c68facb2d8a975b67584432c2b4d57504d99beffe6922e9"
+  license "BSD-3-Clause"
+  head "https://github.com/clangen/musikcube.git", branch: "master"
+
+  depends_on "asio" => :build
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "ffmpeg"
+  depends_on "game-music-emu"
+  depends_on "gnutls"
+  depends_on "lame"
+  depends_on "libev"
+  depends_on "libmicrohttpd"
+  depends_on "libogg"
+  depends_on "libopenmpt"
+  depends_on "libvorbis"
+  depends_on :macos
+  depends_on "ncurses"
+  depends_on "openssl@1.1"
+  depends_on "taglib"
+  uses_from_macos "curl"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    system "make", "install"
```

Comment:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```

Reply:
Yea, it's been on my todo list for a while 😅

Reply:
https://github.com/Homebrew/homebrew-core/pull/118938/commits/b344ea930578a7cf202ca82762ab85bfd399f396